target/linux/ath79/mikrotik/base-files/etc/board.d/02_network

```diff
@@ -24,6 +24,12 @@ ath79_setup_interfaces()
 	mikrotik,routerboard-wapr-2nd)
 		ucidef_set_interface_lan "eth0"
 		;;
+	mikrotik,routerboard-951ui-2nd|\
+	mikrotik,routerboard-952ui-5ac2nd)
+		ucidef_set_interfaces_lan_wan "eth0" "eth1"
```

Comment:
is this correct? I changed it to include eth1 as wan

Reply:
No. You shouldn't need to change anything from my PR. The hAP is identical to hAP ac lite minus the PCI 802.11ac wireless.